Azure/azure-cli
# [AKS] az aks check-acr: Add new check-acr command

**Description**


This PR add a new command `az aks check-acr` to the aks module. This command allows customers to easily check the ACR connection to the AKS cluster. See the testing guide for more details.

**Testing Guide**


Example usage: `az aks check-acr -g myResourceGroup -n myAKSCluster --acr myacr.azurecr.io`

Eample Output:

```
[2021-01-11T21:08:53Z] Checking host name resolution (myacr.azurecr.io): SUCCEEDED
[2021-01-11T21:08:53Z] Canonical name for ACR (myacr.azurecr.io): r1209eus.eastus.cloudapp.azure.com.
[2021-01-11T21:08:53Z] ACR location: eastus
[2021-01-11T21:08:53Z] Loading azure.json file from /etc/kubernetes/azure.json
[2021-01-11T21:08:53Z] The cluster uses service principal.
[2021-01-11T21:08:53Z] Validating service principal credential: SUCCEEDED
[2021-01-11T21:08:53Z] Validating image pull permission: SUCCEEDED
[2021-01-11T21:08:53Z]
Your cluster can pull images from myacr.azurecr.io!
```

**History Notes**


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
